packages/utils/test/lighthouserc.test.js

```diff
@@ -38,40 +45,138 @@ describe('lighthouserc.js', () => {
       });
     });
 
-    it('should convert keys with . in them', () => {
-      const assertions = {'resource-summary.script.size': 'off'};
-      writeFile({ci: {assert: {assertions}}});
-      expect(rc.loadAndParseRcFile(tmpFile)).toEqual({
+    describe('should convert keys with . in them', () => {
+      const parsedContent = {
         assertions: {
           'resource-summary:script:size': 'off',
         },
+      };
+      it('JSON', () => {
+        writeJSONFile(rcJSONFilePath, {
+          ci: {assert: {assertions: {'resource-summary.script.size': 'off'}}},
+        });
+        expect(rc.loadAndParseRcFile(rcJSONFilePath)).toEqual(parsedContent);
       });
-    });
 
-    it('should load and flatten other properties', () => {
-      writeFile({
-        ci: {assert: {x: 1}},
-        'ci:client': {collect: {y: 2}, server: {ignored: true}},
-        'ci:server': {collect: {ignored: true}, server: {z: 3}},
+      it('YAML', () => {
+        writeFile(
+          rcYamlFilePath,
+          `
+        ci:
+          assert:
+            assertions:
+              resource-summary.script.size: 'off'
+        `
+        );
+        expect(rc.loadAndParseRcFile(rcYamlFilePath)).toEqual(parsedContent);
       });
+    });
 
-      expect(rc.loadAndParseRcFile(tmpFile)).toEqual({
+    describe('should load and flatten other properties', () => {
+      const parsedContent = {
         x: 1,
         y: 2,
         z: 3,
+      };
+      it('JSON', () => {
+        writeJSONFile(rcJSONFilePath, {
+          ci: {assert: {x: 1}},
+          'ci:client': {collect: {y: 2}, server: {ignored: true}},
+          'ci:server': {collect: {ignored: true}, server: {z: 3}},
+        });
+        expect(rc.loadAndParseRcFile(rcJSONFilePath)).toEqual(parsedContent);
+      });
+      it('YAML', () => {
+        writeFile(
+          rcYamlFilePath,
+          `
+        ci:
+          assert:
+            x: 1
+        ci:client:
+          collect:
+            y: 2
+          server:
+            ignored: true
+        ci:server:
+          collect:
+            ignored: true
+          server:
+            z: 3
+        `
+        );
+        expect(rc.loadAndParseRcFile(rcYamlFilePath)).toEqual(parsedContent);
       });
     });
   });
 
   describe('#findRcFile', () => {
     const LH_ROOT = path.join(__dirname, '../../../');
+    const FIXTURES_DIR = path.join(__dirname, 'fixtures');
 
-    it('should find an rcfile in the directory', () => {
-      const expected = path.join(LH_ROOT, 'lighthouserc.json');
-      expect(rc.findRcFile(LH_ROOT)).toEqual(expected);
+    describe('should find an rcfile in the directory', () => {
+      it('.json', () => {
+        const expected = path.join(LH_ROOT, 'lighthouserc.json');
+        expect(rc.findRcFile(LH_ROOT)).toEqual(expected);
+      });
+      it('.yaml', async () => {
+        const expected = path.join(FIXTURES_DIR, 'lighthouserc.yaml');
+        writeFile(expected, '');
```

Comment:
can we use a tmp directory for these if we're writing to them in tests? don't want to worry about commiting them later

Reply:
Great suggestion.

Created a temp dir for all files.